exo-go/src/process_helpers/index.go

```diff
@@ -38,6 +40,17 @@ func Start(dir string, commandWords ...string) (*exec.Cmd, io.WriteCloser, *byte
 	return cmd, in, &out, nil
 }
 
+// Wait waits for the given text
+func Wait(stdout fmt.Stringer, text string, done func()) {
+	ticker := time.NewTicker(20 * time.Millisecond)
```

Comment:
reduced the tick time, still waiting for @charlierudolph to respond to my questions about how to use pipes.